### ISSUE TITLE
How To Hide Related Videos?

### ISSUE THREAD

**Issue by cyruslk** [tjallingt/react-youtube#297]
Is there a way to add related videos that appear on pause?

Thanks a lot in advance and immense gratitude for this package.

**Comment by michaeltnguyen** [comment]
I don't believe this is possible: https://developers.google.com/youtube/player_parameters#rel

> you will not be able to disable related videos. Instead, if the rel parameter is set to 0, related videos will come from the same channel as the video that was just played.

**Comment by Abdelalim-dev** [comment]
As @michaeltnguyen said, you cannot disable them.
They used to be disable-able in the past but Google changed that. Now you can only limit the related videos to the same channel of the video playing (rel=0)

**Comment by staticGuru** [comment]
YouTube's default UI elements can be hidden, related videos can be hidden, and ads can be removed from YouTube videos in react native.

It may help you to meet your requirements if you use the below-mentioned package:
https://www.npmjs.com/package/react_native_youtube_streamer

Besides being open source, the package is scalable as well. It is also possible for you to contribute to that project.

**Comment by PetersonFonseca** [comment]
opts={{
  playerVars: {
    controls: 0,
     rel: 0
   }
}}
